build.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    if std::path::Path::new("/dev/sev").exists() {
```

Comment:
Does this means we must build on a hardware platform with sev capability?
Does it work by simple define a feature flag?

Reply:
No, this is only required to avoid executing tests and examples on platforms that do not support SEV